pkg/cmd/admin/registry/registry.go

```diff
@@ -299,6 +302,9 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	env := app.Environment{}
 	env.Add(secretEnv)
 
+	if cfg.EnforceQuota {
```

Comment:
Defaulting the flag to true and overriding the envvar means you cannot disable with the envvar alone